pages/articles/building-docu.md

```diff
@@ -0,0 +1,35 @@
+Docker with Linux/Mac OS/WSL:
+
+Build documentation
+
+```bash
+docker run --rm -it -v $(pwd):/home mono:6.12.0.182 sh -c "apt update && apt upgrade -y && apt install wget unzip -y; \
+    wget https://github.com/dotnet/docfx/releases/download/v2.51/docfx.zip; \
+    unzip docfx.zip -d /usr/local/lib/docfx; \
+    chmod +x /usr/local/lib/docfx/docfx.exe; \
+    /usr/bin/mono /usr/local/lib/docfx/docfx.exe build /home/pages/docfx.json;"
+```
+
+Clean documentation
```

Comment:
Same here as well , can you add it as a heading text ?? Just add `##` before this section for better readability